src/components/Certify/Certify.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <v-card flat id="AR-step-4-container">
+    <div class="certify-container">
+      <div class="certifiedby-container">
+        <label>Legal Name</label>
+        <div class="value certifiedby">
+          <v-text-field
+            filled
+            persistent-hint
+            id="certified-by-textfield"
+            label="Person's legal name"
+            hint="Enter the legal name of the person authorized to complete and submit these changes."
```

Comment:
I know you just changed this, but now that I'm looking at the design it doesn't match.
https://preview.uxpin.com/c53183da94b200338185d9557b11dd9a860f9092#/pages/131426862/simulate/no-panels

Do you have a later design that asks for the wording above?